src/main/java/uk/nhs/digital/nhsconnect/nhais/exceptions/SenderValidationException.java

```diff
@@ -0,0 +1,7 @@
+package uk.nhs.digital.nhsconnect.nhais.exceptions;
+
+public class SenderValidationException extends RuntimeException {
```

Comment:
Can we use one of the subclasses of NhaisBaseException so that a correct OperationOutcome error is returned by the API?